doc/usage.md

```diff
@@ -102,7 +102,7 @@ $ echo '{"jsonrpc": "2.0", "method": "server.version", "params": ["", "1.4"], "i
 ```
 
 For more complex tasks, you may need to convert addresses to 
-[script hashes](https://electrumx-spesmilo.readthedocs.io/en/latest/protocol-basics.html#script-hashes) - see 
+[script hashes](https://electrumx-spesmilo.readthedocs.io/en/latest/protocol.html#script-hashes) - see 
```

Comment:
Let's move to https://electrum-protocol.readthedocs.io/ (generated from https://github.com/spesmilo/electrum-protocol):
```suggestion
[script hashes](https://electrum-protocol.readthedocs.io/en/latest/protocol-basics.html#script-hashes) - see 
```

Reply:
Thanks!
Could you please also update the rest of the URLs?

https://github.com/search?q=repo%3Aromanz%2Felectrs%20electrumx&type=code